### PR TITLE
migrate feature.ts functions

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
Screenshot from 2022-04-25 20-55-58
Screenshot from 2022-04-25 20-57-57
Screenshot from 2022-04-25 20-58-07
Screenshot from 2022-04-25 21-00-24
Screenshot from 2022-04-25 21-00-50
Screenshot from 2022-04-25 21-01-53

### CODE REVIEW

File: packages/odf/features.ts [hunk 1]
```diff
@@ -59,6 +86,16 @@ export const OCS_FEATURE_FLAGS = {
...
 
+export const ODF_BLOCK_FLAG = {
+  [FEATURES.SS_LIST]: 'ss-list',
+  [FEATURES.ADD_CAPACITY]: 'add-capacity',
+  [FEATURES.ODF_WIZARD]: 'install-wizard',
+  [FEATURES.BLOCK_POOL]: 'block-pool',
+  [FEATURES.MCG_RESOURCE]: 'mcg-resource',
+  [FEATURES.ODF_DASHBOARD]: 'odf-dashboard',
+  [FEATURES.COMMON_FLAG]: 'common',
+};
```
Comment (bipuladh) [inline packages/odf/features.ts:97]: Why are we blocking these stuffs?

File: packages/odf/features.ts [hunk 2]
```diff
@@ -113,3 +150,179 @@ export const setOCSFlags = async (setFlag: SetFeatureFlag) => {
...
+  csv: ClusterServiceVersionKind
+) => {
+  const support = JSON.parse(getAnnotations(csv)?.[OCS_SUPPORT_ANNOTATION]);
+  _.keys(OCS_FEATURE_FLAGS).forEach((feature) => {
+    setFlag(feature, support.includes(OCS_FEATURE_FLAGS[feature]));
+  });
+  // Todo(bipuladh): Remove array string after CI starts using ODF 4.10
+  const disabled = JSON.parse(
+    getAnnotations(csv)?.[OCS_DISABLED_ANNOTATION] || '[]'
+  );
```
Comment (bipuladh) [inline packages/odf/features.ts:295]: I don't think this is required here.

**Review by bipuladh — CHANGES_REQUESTED** [review]

File: packages/odf/features.ts (hunk 2 above)
Comment (SanjalKatiyar) [reply packages/odf/features.ts]: ack...

**Review by bipuladh — APPROVED** [review]

**Pull request merged**